### ISSUE TITLE
Ping Result not reported properly

### ISSUE THREAD

**Issue by carryel** [Cacti/cacti#5665]
# Describe the bug

cacti version is develop branch.

Ping Result was always "ICMP ping Timed out" even though the server was alive.

When debugging this issue, the parsing of the results appeared to be incorrect.

- PHP_OS was "Linux".
- ping command was "fping -q -t 400 -c 1 -r 1 my.host.com".
- Result was "my.host.com : xmt/rcv/%loss = 1/1/0%, min/avg/max = 0.258/0.258/0.258".

It seems to be a side-effect of https://github.com/Cacti/cacti/commit/011ae4c5c53a65d3a6ff077ed66601d79112af4e#diff-22abb1862abff566f5588ec6a9666d11553d088d2bb54a280a7b21b2ca36b499R225 .

I think the "if statement" should be reversed like this:

`if (strtoupper(substr(PHP_OS, 0, 3)) !== 'WIN') {`

https://github.com/Cacti/cacti/blob/develop/lib/ping.php#L225

## Screenshots
<img width="723" alt="Screenshot 2024-02-07 at 3 08 38 PM" src="https://github.com/Cacti/cacti/assets/382836/3e48058c-b4fc-4c78-8eb4-40ee3d6a8907">
<img width="771" alt="Screenshot 2024-02-07 at 3 08 50 PM" src="https://github.com/Cacti/cacti/assets/382836/bee53a5b-b587-4deb-950b-6ebfce5bf877">

## Desktop (please complete the following information)

- OS: CentOS7

**Comment by dallenk** [comment]
Also experiancing the same results with commits after https://github.com/Cacti/cacti/commit/011ae4c5c53a65d3a6ff077ed66601d79112af4e

Advanced ping template stops graphing all hosts..  git reset --hard 4563aaa1d resolves it for me.

**Comment by arno-st** [comment]
Same here, but didn't do the troubleshooting.
Except that the ping is OK for spine, and in CLI, but on the dipslay it said ping timeout.
I'm using icmp ping and SNMP uptime to get the status and the device is UP as it should be, with the ping timeout error.
Cacti dev branch, 
On almalinux

**Comment by TheWitness** [comment]
Okay, fixed this.  No changelog as this was between releases.  Thanks for reporting all!
